src/settings/components/LoadDataTabbedPage.tsx

```diff
@@ -22,12 +23,15 @@ type StateProps = ConnectedProps<typeof connector>
 
 type Props = ComponentProps & StateProps
 
+@ErrorHandling
```

Comment:
```suggestion
```

Reply:
Not sure why I'm seeing this thread twice.  Double checked that it was removed though :)